Tasks/Romanychev/Task4/text_processor.py

```diff
@@ -0,0 +1,98 @@
+class TextProcessor:
```

Comment:
it's a relatively good usage of OOP, but I will ask you one simple question, and I want you to ask yourself this question each time you want to use OOP - why do you need a class here?